Gemfile

```
@@ -3,7 +3,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '3.2.2'
+ruby '3.3.5'
```

I don't think we should specify a Ruby version here, as `cfonb` is just a dependency, not _final product_: the usual directive in a Gemfile for a gem is `gemspec`.

```suggestion
gemspec
```